pinot-common/src/main/java/com/linkedin/pinot/common/utils/SegmentNameBuilder.java

```diff
@@ -16,13 +16,19 @@
 package com.linkedin.pinot.common.utils;
 
 public class SegmentNameBuilder {
```

Comment:
Please fold in the methods of this class as (protected) methods in DefaultSegmentNameGenerator and remove this class altogether. There are no users of these methiods